jenkins/stretchltp.jpl

```diff
@@ -0,0 +1,13 @@
+@Library('kernelci') _
+import org.kernelci.debian.RootFS
+
+def r = new RootFS()
+
+def config = ['name':"stretchltp",
+              'arch_list':["arm64", "i386", "amd64"],
```

Comment:
How about `armhf`?  I think there are enough platforms around with that arch to justify building it too.